server/routerlicious/packages/lambdas/src/deli/lambdaFactory.ts

```diff
@@ -223,6 +236,7 @@ export class DeliLambdaFactory extends EventEmitter implements IPartitionLambdaF
         ).catch((error) => {
             const errMsg = "Deli Lambda failed to mark session as active.";
             context.log?.error(`${errMsg}. Exception: ${inspect(error)}`, { messageMetaData });
+            Lumberjack.error(`${errMsg}. Exception: ${inspect(error)}`, { messageMetaData });
```

Comment:
Let's extract the error into the `exception` property and use `getLumberBaseProperties`